data/remnant/remnant.txt

```diff
@@ -518,10 +648,60 @@ phrase "friendly remnant"
 	word
 		" "
 	word
-		"They're the best!"
+		"They are the best!"
 		"You should definitely try it!"
 		"Definitely better than those outfits in your space."
 
+phrase "remnant outfit recommendations (trusted)"
+	word
+		"Since"
+	word
+		" "
+	word
+		"you have"
+		"the prefects gave you"
+		"our elders allowed you"
+		"our government granted you"
+	word
+		" "
+	word
+		"a license"
+		"a permit"
+		"permission to equip our outfits"
+	word
+		", "
+	word
+		"I"
+		"our experts"
+		"our veterans"
+		"my friends"
+		"we"
+	word
+		" "
+	word
+		"would suggest"
+		"would recommend"
+	word
+		" "
+	word
+		"equipping"
+		"getting"
+		"installing"
+		"outfitting"
+	word
+		" "
+	phrase
+		"remnant inhibitor cannon"
```

Comment:
Should these be capitalized?

Reply:
No, as that is the name of the phrase, not literally what the player sees.